### PR TITLE
Make the sourced accordion into a stacked accordion

### LINKED ISSUE #3200: Pause panel should use a stacked sidebar accordion
I just landed Ryan's PR which fixes the layout issues for the pause panel, but unintentionally switched the pause panel to the sources accordion ux where the panels are evenly spaced.

Lets bring back the stacked accordions which avoids unnecessary empty space for the top panels

### PR DESCRIPTION
**Pull request by jcmorrow**
This adds a new Accordion class, which shares an interface with the old one, but is IMO considerably simpler. Right now it only has a stacked mode, but it's quite simple to add a non-stacked mode by setting `flex-basis: 0` and `flex-grow: 1` on open children. One thing that I don't love about this design is that the heights use a content basis, which means that if there is any overflow, the shrunken space will be shared evenly by the children, but their final heights will reflect differences in their contents. This means in situations where there are contents with very different heights you will have very uneven heights after shrinking as well. There are a lot of examples in storybook to play around with.

CleanShot 2021-11-16 at 11 44 54

CleanShot 2021-11-16 at 11 44 42

Fixes #3200 (finally)

### CODE REVIEW

**Comment by jasonLaster** [comment]
Nice use case for story book

File: src/devtools/client/debugger/src/components/shared/Accordion.css [hunk 1]
```diff
@@ -68,6 +68,20 @@
...
+  flex-basis: auto;
+  min-height: fit-content;
+}
+
+.accordion li.expanded {
+  min-height: 20%;
+}
+
+.accordion li.lastOpen {
+}
```
Comment (jaril) [inline src/devtools/client/debugger/src/components/shared/Accordion.css:87]: Unused style.

File: src/ui/components/Accordion.module.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.accordionItemContainer:not(.open) {
+  flex-shrink: 0;
+}
+
+.accordionItemContainer .header {
+  background: white;
+  cursor: pointer;
+  font-size: 15px;
+  line-height: calc(16 / 12);
```
Comment (jaril) [inline src/ui/components/Accordion.module.css:32]: Small nit but we can probably get away with just `1.25`.

**Review by jaril — APPROVED** [review]

File: src/ui/components/Accordion.module.css (hunk 2 above)
Comment (jcmorrow) [reply src/ui/components/Accordion.module.css:32]: Yeah, this was pre-existing, but I agree with you, I'll change it 👍

**Pull request merged**